src/Forms/Main.cs

```diff
@@ -26677,8 +26678,46 @@ private void toolStripMenuItemDost_Click(object sender, EventArgs e)
 
         private void toolStripMenuItemMeasurementConverter_Click(object sender, EventArgs e)
         {
-            var form = new MeasurementConverter();
-            form.Show(this);
+            var selectedText = string.Empty;
+            if (tabControlSubtitle.SelectedIndex == TabControlSourceView)
+            {
+                selectedText = textBoxSource.SelectedText;
+            }
+            else
+            {
+                if (textBoxListViewText.Focused)
+                {
+                    selectedText = textBoxListViewTextAlternate.SelectedText;

```

Comment:
Should be `textBoxListViewText.SelectedText;`  ?

Reply:
Ah, the confusion happened when I switched the conditions to include the case when text is selected but the focus is in the list view.